<launch>
 


   <node pkg="obj_detection" type="VisionNode" name="VisionNode" output="screen" >
	<param name="device" type="int" value="2" />
	<param name="width" type="double" value="960" />
	<param name="height" type="double" value="540" />
	<param name="fps" type="double" value="30" />
	<param name="brigthness" type="double" value="0.219608" />
	<param name="contrast" type="double" value="0.3333333" />
	<param name="saturation" type="double" value="0.501961" />
	<param name="hue" type="double" value="0.5" />
	<param name="gain" type="double" value="0.125" />

	<param name="u0" type="double" value="0" />
	<param name="v0" type="double" value="0" />
	<param name="fu" type="double" value="0" />
	<param name="fv" type="double" value="0" />
	<param name="k1" type="double" value="0" />
	<param name="k2" type="double" value="0" />
	<param name="p1" type="double" value="0" />
	<param name="p2" type="double" value="0" />

	<param name="real_time" type="bool" value="true"/>
	<param name="img_path" type="string" value="/home/ste/Immagini/test_drone/"/>
	<param name="img_path_save" type="string" value="/home/odroid/"/>
	<param name="save_img" type="bool" value="true"/>
	<param name="camera_param" type="string" value="camera.txt"/>
    </node>


        
</launch>
